IPython/core/magics/execution.py

```diff
@@ -71,23 +72,27 @@ class TimeitResult(object):
 
     """
 
-    def __init__(self, loops, repeat, best, worst, all_runs, compile_time, precision):
+    def __init__(self, loops, repeat, average, stdev, all_runs, compile_time, precision):
         self.loops = loops
         self.repeat = repeat
-        self.best = best
-        self.worst = worst
+        self.average = average
+        self.stdev = stdev
```

Comment:
On the other hand because the printed message to stdout ia the same that coils be used to refactor some code and remove redundancy which I think is more than enough reason to move the calculation to the __init__ method.